app/auth_test.go

```diff
@@ -97,6 +97,18 @@ func TestAuthManager(t *testing.T) {
 	code, err = am.AuthToken()
 	assert.Equal(t, client.AuthToken("footoken"), code)
 	assert.NoError(t, err)
+
+	am = NewAuthManager(AuthManagerConfig{
+		AuthDataStore: ms,
+		IdentitySource: &dev.IdentityDataRunner{
+			Cmdr: cmdr,
+		},
+		KeyStore: store.NewKeystore(ms, "key", true),
+	})
+	err = am.GenerateKey()
+	if assert.Error(t, err) {
+		assert.True(t, store.IsStaticKey(err))
+	}
```

Comment:
I don't think you need to to protect the second assert.

```suggestion
assert.Error(t, err)
assert.True(t, store.IsStaticKey(err))
```

Reply:
It makes the error message a little cleaner if the first one fails.